hotp/ta/hotp_ta.c

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (c) 2017, Linaro Limited
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
Replacing this with
```
SPDX-License-Identifier: BSD-2-Clause
```
would save a few lines.

Reply:
Have we adopted SPDX now? Even though I know LinuxFoundation (Kate) has been asking for that quite a while and I don't mind, it'd be good to have a formal decision saying that this should be added to all new files (I can only see it in two files in the entire OP-TEE project).

Reply:
Yep let's use SPDX whenever we can. There are 15 files with a SPDX tag in optee_os master already.